### ISSUE TITLE
Add missing keyboard shortcuts

### ISSUE THREAD

**Issue by aninde** [handsontable/handsontable#8459]
### Description
I was able to find some missing keyboards shortcuts that we should add on the https://handsontable.com/docs/8.2.0/tutorial-keyboard-navigation.html

1.  <kbd>Shift</kbd> + <kbd>Enter</kbd> - opens editor and move to the cell above current active cell (if exists) 
2. <kbd>Ctrl</kbd> + <kbd>Enter</kbd> - opens editor and adds new line in the cell 
3. <kbd>Page Up</kbd> - Move up one screen <br>
4. <kbd>Page Down</kbd> - Move down one screen

### Links
https://handsontable.com/docs/8.2.0/tutorial-keyboard-navigation.html

**Comment by warpech** [comment]
@wszymanski can you confirm if we really support the combinations <kbd>Shift</kbd> + <kbd>Enter</kbd>,<kbd>Ctrl</kbd> + <kbd>Enter</kbd>? I tried them I don't think they work or maybe they work slightly differently than explained in the OP.

**Comment by wszymanski** [comment]
It seems that author of the code tried to provide some features to the combinations:

- `shift` + `enter` should move selection when an editor is closed.

https://github.com/handsontable/handsontable/blob/217086a2f003053e6972e91befdd912c04f491d7/handsontable/src/editorManager.js#L495

- `ctrl` + `enter` should populate values when an editor is opened

https://github.com/handsontable/handsontable/blob/217086a2f003053e6972e91befdd912c04f491d7/handsontable/src/editorManager.js#L478-L482

**Comment by warpech** [comment]
> that author of the code tried to provide some features to the combinations

I read between the lines that these features do not work as intended. 

@aninde is this a regression that needs reporting?

**Comment by kirszenbaum** [comment]
These are my findings on Mac, regarding shortcuts listed in this issue ([here](https://github.com/handsontable/handsontable/issues/8459#issue-944380343) and [here](https://github.com/handsontable/handsontable/issues/8459#issuecomment-756097839)):

| Shortcut | Before editing | When editing |
| --- | --- | --- |
| <kbd>Shift</kbd> + <kbd>Enter</kbd> | Start editing (but **anything** + <kbd>Enter</kbd> starts editing)  |  Save changes, and move one cell up |
| <kbd>Ctrl</kbd> + <kbd>Enter</kbd> | Start editing (but **anything** + <kbd>Enter</kbd> starts editing)  |  Add a new line inside the cell |
| <kbd>Page Up</kbd> | Move one screen up | Save changes, and move one screen up |
| <kbd>Page Down</kbd> | Move one screen down | Save changes, and move one screen down |
| <kbd>Enter</kbd> | Start editing | Save changes, and move one cell down |
| <kbd>F2</kbd> | Start editing | Nothing happens |
| <kbd>Esc</kbd> | Nothing happens | Cancel changes |
| <kbd>Up↑</kbd> | Move one cell up | Move the cursor to the cell's beginning |
| <kbd>Down↓</kbd> | Move one cell to down | Move the cursor to the cell's end |
| <kbd>Right→</kbd> | Move one cell to the right | Move the cursor one sign to the right |
| <kbd>Left←</kbd> | Move one cell to the left | Move the cursor one sign to the left |
| <kbd>Tab</kbd> | Move one cell to the right | Save changes, and move one cell to the right |
| <kbd>Shift</kbd> + <kbd>Tab</kbd> | Move one cell to the left | Save changes, and move one cell to the left |
| <kbd>Home</kbd> | Move to the first cell of the current row | Move the cursor to the cell's beginning |
| <kbd>End</kbd> | Move to the last cell of the current row  | Scrolls the whole page to the bottom |

Based on the above, I'm updating #9271 with [this commit](https://github.com/handsontable/handsontable/pull/9271/commits/9fd98d08c832de8ebf37ea7f34e49b2fe5b8f4c7).

But IMO **all** keyboard shortcuts should be thoroughly tested using different browsers, on both Windows and Mac, **and** then added/removed/updated in the docs.

**Comment by warpech** [comment]
@aninde are all of these covered in the test cases? If not, they should be immediately added and verified. If this must wait until after the release, that's fine. We can release the docs as is (amazing work, @kirszenbaum!) and fix the docs soon after the release if needed.

**Comment by aninde** [comment]
> > that author of the code tried to provide some features to the combinations
> 
> I read between the lines that these features do not work as intended.
> 
> @aninde is this a regression that needs reporting?

This option works when several cells are selected with the `area highlight` class. It works according to Excel.

**Comment by kirszenbaum** [comment]
@warpech @wszymanski @warpech – after the recent developments regarding keyboard shortcuts, this issue is out of date by now. Moving to Done
